include/internal/verifier.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2022 Intel Corporation
+ * Copyright (c) 2020-2021 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _INTERNAL_VERIFIER_H
+#define _INTERNAL_VERIFIER_H
+
+#include <librats/core.h>
+
+#define RATS_VERIFIERS_DIR "/usr/local/lib/librats/verifiers/"
```

Comment:
The install lib path is set(RATS_INSTALL_LIB_PATH "${RATS_INSTALL_PATH}/lib/rats").

RATS_VERIFIERS_DIR should consistent with RATS_INSTALL_LIB_PATH/verifiers/